packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey, EmptyKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+import { TreeSchemaIdentifier } from "../../schema-stored";
+
+export const proxySymbol = Symbol.for("editable-tree");
```

Comment:
I think searching the runtime for other symbols with this string is not useful here. Just use the constructor and then we can also use unique symbol.

We could also use symbols for accessing other data off nodes like the schema/type of the node:

```suggestion
export const proxySymbol: unique symbol = Symbol("editable-tree proxySymbol");
export const type: unique symbol =Symbol("editable-tree type");
```